app/controllers/admin/items_controller.rb

```diff
@@ -4,7 +4,13 @@ def index
   end
 
   def update
-    update_item
+    if params[:order_id] != nil
```

Comment:
Add conditional for when a merchant wants to fulfill just their items in an order